src/metadata/trackmetadatataglib.cpp

```diff
@@ -375,14 +377,14 @@ void writeID3v2TextIdentificationFrame(TagLib::ID3v2::Tag* pTag,
 
     const TagLib::String::Type stringType =
             getID3v2StringType(*pTag, isNumericOrURL);
-    QScopedPointer<TagLib::ID3v2::TextIdentificationFrame> pTextFrame(
-            new TagLib::ID3v2::TextIdentificationFrame(id, stringType));
+    auto pTextFrame(
+            std::make_unique<TagLib::ID3v2::TextIdentificationFrame>(id, stringType));
```

Comment:
IMHO a = in combination with auto is much better to read here. 
It seams that this is also one benefit for std::make_unique, that "=" can be used here without a risk of 
declaring a plain pointer. 
 
```
auto pTextFrame = 
        std::make_unique<TagLib::ID3v2::TextIdentificationFrame>(id, stringType);
```

If you agree, I like to put this as good example into the Mixxx style guide.